SCons/EnvironmentTests.py

```diff
@@ -1004,9 +1004,9 @@ def test_BuilderWrapper_attributes(self):
         e1.Replace(BUILDERS = {'b' : b1})
         bw = e1.b
 
-        assert bw.env is e1
+        assert bw.env == e1
```

Comment:
Did the checkers flag this?
seems like using `is` is different than `==` ? 
Or am I mistaken?

Reply:
There many instances of changing the comparisons in the tests, just curious about the why and if by changing them we could miss some changed functionality?

Reply:
In general, these changed because an identity test wasn't really interesting.  In the case of these two, I'm looking again and it seems like it is. In the others, like l.1839, `assert env2 is env2` isn't meaninful: if an object fails to have the same identity as itself, Python is broken, not SCons.  It's been a while, don't have the checker output any longer... change l.1007-1009 back?

Reply:
Pushed changes, I believe only those two are appropriate as identity checks, the other replacements cause the tests to actually test something useful.